addons/website_slides/populate/slide_channel_partner.py

```diff
@@ -0,0 +1,51 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import random
+from collections import defaultdict
+
+from odoo import models
+from odoo.tools import populate
+
+
+class SlideChannelPartner(models.Model):
+    _inherit = 'slide.channel.partner'
+    _populate_dependencies = ['res.partner', 'slide.channel']
+    # 50% more than partners, so supports 0 to N courses per partner.
+    _populate_sizes = {'small': 150, 'medium': 3_000, 'large': 150_000}
```

Comment:
150_000 with an underscore ?

Reply:
Yes, `150_000` is more readable than `150000`. It's [PEP 515](https://peps.python.org/pep-0515/) in Python 3.6.

Reply:
Going to my "please don't do that" list then.